src/components/ActionPage.jsx

```diff
@@ -0,0 +1,148 @@
+} from "../styled-components/ActionPage";
+import geffroy from "../assets/ActionImg/Geffroy.jpeg";
+import marsac from "../assets/ActionImg/Marsac.jpeg";
+import linkedin from "../assets/icons/linkedin.png";
+
+export default function ActionPage() {
+  return (
+    <DivMain>
+      <H1>Expérimentons ensemble !</H1>
+      <DivInfos>
+        <DivCards>
+          <H2>En savoir plus sur nos interventions</H2>
+          <P>
+            Vous souhaitez avoir accès au centre de ressources Favart :
+            présentation institutionnelle, retours d’expériences, présentation
+            détaillée de nos 4 ingénieries, ...
+          </P>
+          <DivCenter>
+            <Button>Nous rejoindre</Button>
+          </DivCenter>
+        </DivCards>
+        <DivCardMiddle>
+          <H2MiddleCard>Agissons ensemble</H2MiddleCard>
+          <Ul>
+            <Li>
+              - Contribuer à nos missions aux Appels à Contributions ouverts sur
+              nos engagements actuels et aux Appels à Manifestation d’Intérêt
+            </Li>
+            <Li>
+              - Appels à Projets portés par nos partenaires et d’autres acteurs
+              engagés
+            </Li>
+            <Li>- Engager une action commune avec Favart</Li>
+            <Li>- Solliciter l’engagement de Favart dans vos dynamiques</Li>
+          </Ul>
+          <ButtonCardMiddle>Nous rejoindre</ButtonCardMiddle>
+        </DivCardMiddle>
+        <DivCards>
+          <H2>Echangeons ensemble</H2>
+          <P>
+            Vous souhaitez échanger avec nous d’un projet, d’un enjeu, d’une
+            idée.Contactez-nous et prenons le temps de partager ce qui vous
+            anime.
+          </P>
+          <DivCenter>
+            <ButtonFlash>Rendez-vous flash</ButtonFlash>
+          </DivCenter>
+        </DivCards>
+      </DivInfos>
+      <H1>Notre équipe</H1>
+      <CardFlex>
+        <Card>
+          <ImgContainer src={geffroy} alt="#" />
+          <Container>
+            <H4>Yoann Geffroy</H4>
+            <P>Directeur-Fondateur</P>
+            <P>
+              Yoann est fondateur et directeur de Favart depuis 2009. <br />{" "}
+              Expert de l’innovation sociale et du développement de projets à
+              fort impact sociétal, Yoann intervient dans la mobilisation du
+              droit à l’expérimentation territorial, la construction de nouveaux
+              modèles économiques basés sur les coûts évités et l’ingénierie des
+              outils juridiques hybrides en faveur de projets à fort impact
+              (Apprentis d’Auteuil, Foncière Chênelet, Malakoff Humanis,
+              Fondation Rexel …).
+            </P>
+            <DivLogo>
+              <a href="https://www.linkedin.com/in/favart/ ">
+                <ImgLogo src={linkedin} alt="logo linkedin" />
+              </a>
+            </DivLogo>
+          </Container>
+        </Card>
+
+        <Card>
+          <ImgContainer src={marsac} alt="#" />
```

Comment:
Add a meaningful value to the alt attribute ("Mr Marsac", for example): it should describe the content of your picture